packages/core/src/ui/UIImage.ts

```diff
@@ -0,0 +1,283 @@
+import { Color, MathUtil } from "@galacean/engine-math";
+import { Sprite, SpriteDrawMode, SpriteTileMode } from "../2d";
+import { ISpriteAssembler } from "../2d/assembler/ISpriteAssembler";
+import { SimpleSpriteAssembler } from "../2d/assembler/SimpleSpriteAssembler";
+import { SlicedSpriteAssembler } from "../2d/assembler/SlicedSpriteAssembler";
+import { TiledSpriteAssembler } from "../2d/assembler/TiledSpriteAssembler";
+import { SpriteModifyFlags } from "../2d/enums/SpriteModifyFlags";
+import { Entity } from "../Entity";
+import { RenderQueueFlags } from "../RenderPipeline/BasicRenderPipeline";
+import { BatchUtils } from "../RenderPipeline/BatchUtils";
+import { RenderContext } from "../RenderPipeline/RenderContext";
+import { SubRenderElement } from "../RenderPipeline/SubRenderElement";
+import { assignmentClone, deepClone, ignoreClone } from "../clone/CloneManager";
+import { UIRenderer, UIRendererUpdateFlags } from "./UIRenderer";
+import { UITransform } from "./UITransform";
+import { CanvasRenderMode } from "./enums/CanvasRenderMode";
+
+export class UIImage extends UIRenderer {
+  @deepClone
+  private _color: Color = new Color(1, 1, 1, 1);
+  @ignoreClone
+  private _sprite: Sprite = null;
+  @ignoreClone
+  private _drawMode: SpriteDrawMode;
+  @assignmentClone
+  private _assembler: ISpriteAssembler;
+  @assignmentClone
+  private _tileMode: SpriteTileMode = SpriteTileMode.Continuous;
```

Comment:
The type SpriteTileMode in private _tileMode: SpriteTileMode = SpriteTileMode.Continuous; can be omitted